Tools/autotest/default_params/copter.parm

```diff
@@ -1,4 +1,3 @@
-EK2_ENABLE      1
```

Comment:
shouldn't we replace this by EK3_ENABLE 1 just to inform people that EKF3 is default now ?

Reply:
No, I don't think we need to set parameters that are already the default.